augur/api/view/server/User.py

```diff
@@ -1,276 +0,0 @@
-from flask_login import UserMixin
```

Comment:
User.py was the view-side wrapper for the User endpoints on the backend. Since the frontend and the backend now exist in the same app, this wrapper is no longer used.